src/app/room/room.component.html

```diff
@@ -16,4 +16,6 @@
   <opponents></opponents>
 </div>
 
-<button mat-raised-button (click)="leaveRoom()" class="top-spacer">Leave the room</button>
+<spinner [loadingStatus]="">
```

Comment:
I don'T udnerstand why this is bound to an empty string. Did you mean the `loadingStatus` of the component?